FreeRTOS/FreeRTOS-Plus-TCP
# Fix build issues for the DNS parser unit tests

Description
-----------
This PR fixes the build issues for the DNS parser unit tests.

Test Steps
-----------
```
cmake -S test/unit-test -B test/unit-test/build/ 
make -C test/unit-test/build/ all 
cd test/unit-test/build/
ctest -E system --output-on-failure --verbose
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
